client/components/tinymce/plugins/simple-payments/dialog/index.jsx

```diff
@@ -107,6 +113,10 @@ class SimplePaymentsDialog extends Component {
 		onComplete( null, formErrors );
 	}
 
+	handleChangeTabs = activeTab => {
+		this.setState( { activeTab } );
+	};
+
 	handleSelectedChange = selectedPaymentId => {
```

Comment:
I'm aware that it isn't the primary purpose of this PR but we could do the same here that above.